.github/workflows/CI.yml

```diff
@@ -3,43 +3,63 @@ name: CI
 on: [pull_request, push]
 
 jobs:
-  build:
+  build-pdf:
```

Comment:
We'll need to update the repo settings to amend the "required" builds before merging the PR. @Daltz333